tests/toolbox_image.sh

```diff
@@ -159,6 +230,25 @@ test_system_update_entrypoint()
     test -x "${rootfs_dir}${SYSTEM_UPDATE_ENTRYPOINT}"
 }
 
+test_execute_resize_partition_grow_rootfs_ok()
+{
+    userdata_size="$((STORAGE_DEVICE_END - USERDATA_START))"
+    new_userdata_size="$((userdata_size / 2))"
+    new_userdata_start="$((USERDATA_START + new_userdata_size))"
+
+    sed -i "s/p3.*type/p3 : start=     ${new_userdata_start}, size=     ${userdata_size}, type/" \
+        "${rootfs_dir}${PARTITION_TABLE_FILE}"
+
+    sha512sum "${rootfs_dir}${PARTITION_TABLE_FILE}" > "${rootfs_dir}${PARTITION_TABLE_FILE}.sha512" || return 1
```

Comment:
i kind of started to agree with your use of return 1 as documentation. Maybe if we ever get time to do EMP-374 this will be more apparant as well